MQ.py

```diff
@@ -54,12 +54,9 @@
 class File(object):
     def __init__(self, file):
         self.name = file['Name']
-        self.start = int(file['Start'], 16)
-        self.end = int(file['End'], 16)
-        try:
-            self.remap = file['RemapStart']
-        except KeyError:
-            self.remap = None
+        self.start = int(file.get('Start', '0'), 16)
+        self.end = int(file.get('End', str(self.start)), 16)
```

Comment:
What's the default we want if `'End'` is missing? This will convert start to dec and then interpret it as hex...

Reply:
The default we want is self.start (basically default to size 0). I guess this could indeed be an issue with the correct code, although there are currently no cases of 'Start' present with 'End' missing.
I'll change it to something simpler.